packages/core/src/html/NodeProcessor.js

```diff
@@ -667,9 +667,44 @@ class NodeProcessor {
   }
 
   /*
-   * API
+   * Adds attribute "type=application/javascript" to <script> tag.
```

Comment:
Let's move these related functionalities to a separate file

Reply:
I have shifted the two functions to another file but I'm not sure what would be an appropriate name for the new file. I've named it literally as `scriptAndStyleTagProcessor.js` for now which can be quite 'limiting' but I think it should work for now 😅